Tests/SwerverTests/ResponseSpec/ResponseFormatterSpec.swift

```diff
@@ -0,0 +1,35 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class ResponseFormatterSpec: QuickSpec {
+
+    override func spec() {
+        describe("A ResponseFormatter") {
+            var formatter: ResponseFormatter!
+            var responseBuilder: ResponseBuilder!
+            
+            beforeEach {
+                formatter = ResponseFormatter()
+                responseBuilder = ResponseBuilder()
+            }
+            
+            it("formats a Response object into a String object") {
+                let aResponse = responseBuilder
+                                    .setStatusCode(statusCode: StatusCode.ok.rawValue)
```

Comment:
This indentation seems off. I would only indent it four spaces from the previous line.